src/features/game/save/GameSaveTypes.ts

```diff
@@ -20,6 +20,8 @@ export type FullSaveState = {
  * @prop {string[]} completedObjectives - list of objectives that have been completed by player
  * @prop {string[]} triggeredInteractions - list of itemIds that have been triggered by player
  * @prop {string[]} triggeredActions - list of actions that have been triggered by player
+ * @prop {string[]} attemptedQuizzes - list of quiz ids that have been attempted by player
+ * @prop {string[]} completedQuizzes - list of quiz ids that have been completed by player
```

Comment:
This documentation does not match the type below? This suggests two fields `attemptedQuizzes` and `completedQuizzes` but the type below only has `quizScores`?

Reply:
Comment addressed: commit e005eb73b47901010f4aefab9f3caca4ae83332a